src/sonic_ax_impl/mibs/ietf/rfc4363.py

```diff
@@ -62,7 +62,7 @@ def update_data(self):
             ent = self.db_conn.get_all(mibs.ASIC_DB, s, blocking=True)
             # Example output: oid:0x3a000000000608
             bridge_port_id = ent[b"SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID"][6:]
-            if bridge_port_id not in self.if_bpid_map:
+            if not self.if_bpid_map or bridge_port_id not in self.if_bpid_map:
```

Comment:
Is if_bpid_map returned as {} in reinit_data() if the map is not populated during init?

Reply:
yes. thanks! it's fixed in the py-swsssdk repository!